dhis2-android-app/src/main/java/org/dhis2/mobile/processors/OfflineDataProcessor.java

```diff
@@ -107,7 +111,10 @@ private static void uploadOfflineReports(Context context) {
                         description = ImportSummariesHandler.getDescription(resp.getBody(),
                                 context.getString(R.string.import_failed));
                     }
-                    String message = String.format("(%s) %s", info.getPeriodLabel(), info.getFormLabel());
+
```

Comment:
@idelcano this code for log mixed with other logic is a little dirty, can you extract a separate method or class?